.travis.yml

```diff
@@ -5,9 +5,11 @@ python:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libdb-dev
+  - mkdir wheelhouse
+  - wget 'http://www.well.ox.ac.uk/~jk/wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl' -O 'wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl'
```

Comment:
@jeromekelleher or I could just squash and commit the existing changes since they seem to be working; your call.